ocs_ci/ocs/ui/base_ui.py

```diff
@@ -1485,20 +1500,1136 @@ def __init__(self):
         DataFoundationTabBar.__init__(self)
 
 
-class TopologyTab(DataFoundationDefaultTab):
+class TopologySidebar(BaseUI):
```

Comment:
Did you try to import from class/function?
```
class X():
   from y import z

```

```
func X():
   from y import z

```

Reply:
would not help in this case because you can not do it in class inheritance description
`class StorageSystemDetails(StorageSystemTab)` 

I will solve the problem with next PR, I just don't want to do so much refactoring in XXL PR and dont want to affect existed tests stability